tests/variantstudy/model/command/test_create_link.py

```diff
@@ -258,15 +250,17 @@ def test_create_diff(command_context: CommandContext):
     other_match = CreateLink(
         area1="foo",
         area2="bar",
-        parameters={"hurdles-cost": "true"},
```

Comment:
It doesn't work anymore with these values anymore ? If so it's an issue as R scripts will continue calling the command with this writing.

Reply:
I set it back and everything is fine